SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+The Dockstore system routinely receives security updates to the most recently
```

Comment:
"most recently released tagged minor version" maybe, otherwise it seems a bit redundant?

Reply:
Added